### ISSUE TITLE
Double space with backticks in descriptions

### ISSUE THREAD

**Issue by owenvoke** [BrainMaestro/tldr-php#3]
Double space when cyan coloured with `<fg=cyan>  $1</>` also affects when there are backticks in the description. This isn't really an issue, but looks slightly odd.

https://github.com/BrainMaestro/tldr-php/blob/f7783297c56e74828e880b9d85819f3d64accb35/src/Page.php#L90

I guess something like this resolves it, but doesn't seem like a good workaround.

```php
// color all text in example command usage
'/^`(.+)`/m' => '<fg=cyan>  $1</>',
'/`(.+)`/' => '<fg=cyan>$1</>',
```

![Example screenshot](https://image.prntscr.com/image/ECYQ8hyjThSrxHiqSM_NFA.png)

**Comment by BrainMaestro** [comment]
What command did this occur in?

**Comment by owenvoke** [comment]
```sh
tldr curl
```

The description:
```
Send form-encoded data (POST request of type   application/x-www-form-urlencoded):
```

**Comment by BrainMaestro** [comment]
Okay, seen. I have a quick fix, want to take a look and see if there are any issues that it causes? The truth is that regex is probably not the right way to accomplish this

**Comment by owenvoke** [comment]
Sure. I'll take a look in a little bit.
